compiler-cli/src/main.rs

```diff
@@ -198,7 +199,7 @@ enum Command {
 
     /// Add new project dependencies
     Add {
-        /// The names of Hex packages to add
+        /// The names of packages to add
```

Comment:
```suggestion
        /// The names of Hex packages to add
```

This one is specifically Hex packages